config/site_settings.yml

```diff
@@ -2032,9 +2032,6 @@ developer:
   enable_safe_mode:
     default: true
     client: true
-  enable_experimental_sidebar_hamburger:
```

Comment:
Note I'm not dropping any overrides in the DB in this PR yet since I want to be safe